qiskit/transpiler/passes/optimization/collect_multiqubit_blocks.py

```diff
@@ -109,8 +109,8 @@ def collect_key(x):
             """special key function for topological ordering.
             Heuristic for this is to push all gates involving measurement
             or barriers, etc. as far back as possible (because they force
-            blocks to end). After that, we process gates in order of lowest
-            number of qubits acted on to largest number of qubits acted on
+            blocks to end). After that, we process gates in order of the lowest
+            number of qubits acted on to the largest number of qubits acted on
```

Comment:
This actually was ok before (compare: "sort in order of lowest to highest", which might be a bit easier to see is standard terminology), but the new form is also fine.

Reply:
reverted in 2d6826e0a95f4cc7be2e5b553e0e81a2af056ac3